src/backend/distributed/master/master_delete_protocol.c

```diff
@@ -259,14 +261,10 @@ master_drop_sequences(PG_FUNCTION_ARGS)
 		appendStringInfo(dropSeqCommand, " %s", TextDatumGetCString(sequenceText));
 	}
 
-	dropSuccessful = ExecuteRemoteCommand(nodeName, nodePort, dropSeqCommand);
-	if (!dropSuccessful)
-	{
-		ereport(WARNING, (errmsg("could not delete sequences from node \"%s:" INT64_FORMAT
-								 "\"", nodeName, nodePort)));
-	}
+	SendCommandToWorkers(ALL_WORKERS, DISABLE_DDL_PROPAGATION);
```

Comment:
Yes, add `DISABLE_DDL_PROPAGATION ` and `dropSeqCommand->data` into a list, and have a function `SendCommandListToWorkers (List *)` which simply iterates over the list can calls `SendCommandToWorkers `.

You can decide whether to add such a function or not.